sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -66,89 +53,73 @@ export interface EventData {
 }
 
 // @public
-export class EventDataBatch {
-    // Warning: (ae-forgotten-export) The symbol "ConnectionContext" needs to be exported by the entry point index.d.ts
-    // 
+export interface EventDataBatch {
     // @internal
-    constructor(context: ConnectionContext, maxSizeInBytes: number, partitionKey?: string);
     readonly batchMessage: Buffer | undefined;
     readonly count: number;
+    readonly maxSizeInBytes: number;
     // @internal
     readonly _messageSpanContexts: SpanContext[];
-    readonly partitionKey: string | undefined;
+    readonly partitionId?: string;
+    readonly partitionKey?: string;
     readonly sizeInBytes: number;
     tryAdd(eventData: EventData, options?: TryAddOptions): boolean;
 }
 
 // @public
 export interface EventHubClientOptions {
-    dataTransformer?: DataTransformer;
     retryOptions?: RetryOptions;
     userAgent?: string;
-    webSocket?: WebSocketImpl;
-    webSocketConstructorOptions?: any;
+    webSocketOptions?: WebSocketOptions;
 }
 
 // @public
 export class EventHubConsumerClient {
-    constructor(connectionString: string, options?: EventHubClientOptions);
-    constructor(connectionString: string, eventHubName: string, options?: EventHubClientOptions);
-    constructor(host: string, eventHubName: string, credential: TokenCredential, options?: EventHubClientOptions);
+    constructor(consumerGroup: string, connectionString: string, options?: EventHubClientOptions);
+    constructor(consumerGroup: string, connectionString: string, eventHubName: string, options?: EventHubClientOptions);
+    constructor(consumerGroup: string, fullyQualifiedNamespace: string, eventHubName: string, credential: TokenCredential, options?: EventHubClientOptions);
     close(): Promise<void>;
-    static defaultConsumerGroupName: string;
-    getPartitionIds(): Promise<string[]>;
+    static defaultConsumerGroup: string;
+    getEventHubProperties(options?: GetEventHubPropertiesOptions): Promise<EventHubProperties>;
+    getPartitionIds(options?: GetPartitionIdsOptions): Promise<string[]>;
     getPartitionProperties(partitionId: string, options?: GetPartitionPropertiesOptions): Promise<PartitionProperties>;
-    getProperties(options?: GetPropertiesOptions): Promise<EventHubProperties>;
-    subscribe(consumerGroupName: string, options: SubscriptionOptions): Subscription;
-    subscribe(consumerGroupName: string, partitionId: string, options: SubscriptionOptions): Subscription;
-    subscribe(consumerGroupName: string, partitionManager: PartitionManager, options: SubscriptionOptions): Subscription;
-}
-
-// @public
-export interface EventHubConsumerOptions {
-    ownerLevel?: number;
-    retryOptions?: RetryOptions;
-    trackLastEnqueuedEventInfo?: boolean;
+    subscribe(handlers: SubscriptionEventHandlers, options?: SubscriptionOptions): Subscription;
```

Comment:
Should `SubscriptionOptions` be `SubscribeOptions`?
In other libraries, the guideline we follow is to prefix "Options" with the name of the method

Or should this be `ReceiveOptions`? Because the options are related to the receiver and not the subscription?

cc @bterlson

Reply:
Renamed to `SubscribeOptions` for consistency with other operations (and other packages)